server.js

```diff
@@ -85,21 +85,55 @@ const parseQueryString = (q, body, options = {}) => {
     "only",
     "autoCompositions",
     "gitBranch",
-    "active",
```

Comment:
Don't know why this attribute was added.

Reply:
This is used when publishing the BOM to Dependency Track. 

It controls if the DTrack project is active or inactive. 

We use it heavily

Reply:
Actually, checking now, looks like it's unused.  Apologies

Reply:
I can't recall why that was added during https://github.com/CycloneDX/cdxgen/pull/751 🤔

Reply:
No issues. Thank you for confirming!